* Remove all pointless RM calls as this only results in rebuilding in
* src_install(). -JeR


--- a/Imakefile	2005-05-09 14:46:24.000000000 +0200
+++ b/Imakefile	2010-09-17 02:07:22.000000000 +0200
@@ -128,10 +128,8 @@
 depend:: lex.c gram.c deftwmrc.c lastmake.c vtwm.man
 
 all::
-	$(RM) deftwmrc.* lastmake.*
 
 install::
-	$(RM) deftwmrc.* lastmake.*
 	$(MAKE) install.man
 
 clean::
@@ -144,7 +142,6 @@
 	$(MV) y.tab.h gram.h
 
 deftwmrc.c: system.vtwmrc
-	$(RM) $@
 	echo '/* ' >>$@
 	echo ' * This file is generated automatically from the default' >>$@
 	echo ' * VTWM bindings file system.vtwmrc.'$(SYS_VTWMRC_LOOK)' by the VTWM Makefile.' >>$@
@@ -155,7 +152,6 @@
 	echo '(char *)0 };' >>$@
 
 lastmake.c:
-	$(RM) $@
 	echo '/* ' >>$@
 	echo ' * This file is generated automatically by the VTWM Makefile.' >>$@
 	echo ' */' >>$@
@@ -166,11 +162,9 @@
 	echo '    "" };' >>$@
 
 system.vtwmrc:
-	$(RM) $@
 	$(CP) $@.$(SYS_VTWMRC_LOOK) $@
 
 vtwm.man:
-	$(RM) $@
 	$(LN) doc/$@ $@
 
 XCOMM Handy for developers to check the man page
